airflow/utils/callback_requests.py

```diff
@@ -56,14 +73,36 @@ class TaskCallbackRequest(CallbackRequest):
     def __init__(
         self,
         full_filepath: str,
-        simple_task_instance: SimpleTaskInstance,
+        simple_task_instance: "SimpleTaskInstance",
         is_failure_callback: Optional[bool] = True,
         msg: Optional[str] = None,
     ):
         super().__init__(full_filepath=full_filepath, msg=msg)
         self.simple_task_instance = simple_task_instance
         self.is_failure_callback = is_failure_callback
 
+    def execute(self, dagbag: "DagBag", *, session: Session) -> None:  # pylint: disable=unused-argument
```

Comment:
Yeah previously this was a conscious effort to only pass namedTuple ( which we  converted to simple objects/clases) to send to multiprocessingng PIPES.

Reply:
However, if this is working fine -- no issue with that :)